.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
```

Comment:
I can see we have `gh-pages` branch but no step for publishing docs. We could do similar thing as [here](https://github.com/snowplow-incubator/scala-lru-map/blob/e60cd3b63191fe7f2c40b2da6552d1b04664b07d/.github/workflows/release.yml#L15), so use `sbt makeSite` and then use action to deploy to branch. Related [issue](https://github.com/snowplow-incubator/scala-lru-map/issues/53)